core/networking/peerstore_test.go

```diff
@@ -0,0 +1,39 @@
+package networking_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/networking"
+
+	p2ppeer "github.com/libp2p/go-libp2p-core/peer"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_NewPeerstore(t *testing.T) {
+	store, cleanup := cltest.NewStore(t)
+	defer cleanup()
+
+	db := store.DB.DB()
+
+	peerstore, err := networking.NewPeerstore(db)
```

Comment:
Try creating a new peer store that reads from dB only to avoid reading from memory